tests/README.md

```diff
@@ -146,7 +146,111 @@ with the `zfs-tests.sh` wrapper script will look something like this:
     Results Summary
     SKIP	  52
     PASS	 1129
-    
+
     Running Time:	02:35:33
     Percent passed:	95.6%
     Log directory:	/var/tmp/test_results/20180515T054509
+
+### 4) Example of adding and running test-case (zpool_example)
+
+  This broadly boils down to 5 steps
+  1. Create/Set password-less sudo for user running test case.
+  2. Edit configure.ac, Makefile.am appropriately
+  3. Create/Modify .run files
+  4. Create actual test-scripts
+  5. Run Test case
+
+  Will look at each of them in depth.
+
+  * Set password-less sudo for 'Test' user as test script cannot be run as root
+  * Edit file **configure.ac** and include line under AC_CONFIG_FILES section
+    ~~~~
+      tests/zfs-tests/tests/functional/cli_root/zpool_example/Makefile
+    ~~~~
+  * Edit file **tests/runfiles/Makefile.am** and add line *zpool_example*.
+    ~~~~
+      pkgdatadir = $(datadir)/@PACKAGE@/runfiles
+      dist_pkgdata_DATA = \
+        zpool_example.run \
+        common.run \
+        freebsd.run \
+        linux.run \
+        longevity.run \
+        perf-regression.run \
+        sanity.run \
+        sunos.run
+    ~~~~
+  * Create file **tests/runfiles/zpool_example.run**. This defines the most
+    common properties when run with test-runner.py or zfs-test.sh.
```

Comment:
```suggestion
    common properties when run with test-runner.py or zfs-tests.sh.
```

Reply:
Yes. I will update this over the weekend.

Reply:
Done